content/_index.md

```diff
@@ -48,6 +48,7 @@ Ready to get started? [Click here](getting-started).
 			</div>
 			<div class="col col-auto">
 				<button class="btn btn-secondary playground-btn-flash" disabled>Download binary</button>
+				<a href="/tour/">Tour of TinyGo</a>
```

Comment:
I would add this at the bottom of this file something like this:

```markdown
{{% blocks/feature icon="fa fa-map-signs" title="Tour of TinyGo" url="/tour/" %}}
Take a tour of TinyGo online and try it out.
{{% /blocks/feature %}}
```

Reply:
Unfortunately that doesn't quite work: it looks like that section is only made for 3 features and with 4 it will wrap around:

![Screenshot_20240616_173018](https://github.com/tinygo-org/tinygo-site/assets/729697/7de47149-16c8-4174-aebb-6951ca226892)

We could do this, but I think it looks weird.

However, I have improved the link slightly with a small change:

![Screenshot_20240616_173101](https://github.com/tinygo-org/tinygo-site/assets/729697/7980c139-4553-4783-afb0-8f4baa70cb97)

Reply:
I was going for something like this (on the https://go.dev homepage) but in my opinion looking better.

![Screenshot_20240616_173316](https://github.com/tinygo-org/tinygo-site/assets/729697/36182028-af83-4434-a18b-32241f10ae87)